12-Capstone/deploy-aws-eb.md

```diff
@@ -128,19 +60,21 @@ In order for the EB CLI to have access to your AWS account you need to provide i
 1. Choose a name for your SSH keypair (or use the default).
 1. Enter a passphrase for your SSH keypair and confirm it.
 
-### Create an Environment
+## Step 6: Create an Environment
+
 Now that we've created an EB Application and setup the EB CLI to allow us to work with it, we need to create an Environment. In the context of EB an Environment represents a particular deployed version of your application, including all of the configuration for load balancers, code versions (i.e. what commit/branch is being used), and connected databases.
 
 This allows us to have a single Application with multiple Environments, for example one for production and one for staging the next release so that QA can test it out before using it in the production Environment.
 
 Run the following command to create your first EB Environment:
+
 ```bash
 $ eb create example-env -db -db.engine postgres -db.i db.t2.micro -i t2.micro --envvars SECRET_KEY_BASE=`bundle exec rake secret`
 ```
 
-This sets up a new Environment using mostly the default options. However, we've explicitly asked for the environment to include a database (specificially a PostgreSQL database), and it opts for using the smallest _instance types_ for both the database and the web app servers.
+This sets up a new Environment using mostly the default options. However, we've explicitly asked for the environment to include a database (specificially a PostgreSQL database), and it opts for using the smallest _instance types_ for both the database and the web app servers.  
 
-Instance type is the term AWS uses to identify which "size" of resource is being requested. For databases the `db.t2.micro` instance type is the smallest possible, and the only one that qualifies in the Free Tier, likewise for web application servers the `t2.micro` instance is the only option for usage in the Free Tier.
+Instance type is the term AWS uses to identify which "size" of resource is being requested. For databases the `db.t2.micro` instance type is the smallest possible, and the only one that qualifies in the Free Tier, likewise for web application servers the `t2.micro` instance is the only option for usage in the Free Tier.  If you are curious about the instance types available, you can read more [here](https://aws.amazon.com/ec2/instance-types/).
 
 When the EB CLI prompts you for a database username and password, keep the default `ebroot` for the username and type `password` for the password.
```

Comment:
This is a nitpick, but using the word "here" as the text of a link is really bad for screenreaders. Link text should be semantic!

```
[AWS supports many different instance types](https://aws.amazon.com/ec2/instance-types/), feel free to read more if you're curious
```